src/app/component/user/components/profile/profile-widget/profile-header/profile-header.component.scss

```diff
@@ -1,153 +1,252 @@
-$bp-smallest: 320px;
-$bp-phone: 576px;
-$bp-tablet: 768px;
-$bp-desktop: 1024px;
-$bp-largest: 1440px;
-
-$font-family-first: montserrat, sans-serif;
+$main-font: 'Montserrat', sans-serif;
+$additional-font: 'Open Sans', sans-serif;
+$background-color: #f5f6f6;
+$main-color: #494a49;
+$grey-color: #839c94;
+$grey-light-color: #6a8079;
+$green-color: #13aa57;
+$green-light-color: rgba(19, 170, 87, 0.12);
+$additional-color: #bebebe;
+
+p {
+  margin: 0;
+}
 
 .profile {
-  background: #f5f7f6;
+  height: 366px;
+  background-color: $background-color;
   position: relative;
 
-  .cards {
-    width: 100%;
+  .edit-icon {
+    width: 20px;
+    height: 20px;
+    background-image: url("~/assets/img/profile/icons/edit-line.svg");
+    position: absolute;
+    top: 20px;
+    right: 16px;
   }
 
-  .details__img {
-    text-align: center;
-    width: 100%;
-    height: 88px;
+  .column {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 104px 24px 24px 36px 32px 50px 90px;
 
-    a {
-      position: absolute;
-      top: -15px;
-      right: -10px;
+    .user-image {
+      width: 76px;
+      height: 76px;
+      margin-top: 20px;
+      justify-self: center;
     }
-  }
 
-  .profile__details {
-    .details__descriptions {
+    .name {
+      font-family: $main-font;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 14px;
+      line-height: 20px;
+      color: $main-color;
+      justify-self: center;
+    }
+
+    .location {
+      font-family: $main-font;
+      font-style: normal;
+      font-weight: 600;
+      font-size: 11px;
+      line-height: 16px;
+      color: $grey-light-color;
+      justify-self: center;
+    }
+
+    .online {
+      display: none;
+    }
+
+    .rate {
+      width: 73px;
+      height: 20px;
+      border-radius: 4px;
+      background-color: $green-light-color;
+      font-family: $additional-font;
+      font-style: normal;
+      font-weight: 600;
+      font-size: 11px;
+      line-height: 15px;
+      color: $green-color;
+      justify-self: center;
       display: flex;
-      margin: 15px 0;
-      flex-direction: column;
+      justify-content: center;
       align-items: center;
-      font-family: $font-family-first;
+    }
 
-      .descriptions__name {
-        font-style: normal;
-        font-weight: 500;
-        font-size: 18px;
-        line-height: 15px;
-        text-align: center;
+    .credo {
+      font-family: $additional-font;
+      font-size: 11px;
+      line-height: 16px;
+      color: $main-color;
+      width: 288px;
+      justify-self: center;
+    }
+
+    .social {
+      display: flex;
+      width: 288px;
```

Comment:
Why is the width fixed?

Reply:
Outer block has fixed width according to mockup and particular alignment of content, so this block has fixed width as well.